.talismanrc

```diff
@@ -0,0 +1,2 @@
+allowed_patterns:
```

Comment:
I think our practice is to ignore the `.talismanrc` file instead of committing it.

Reply:
Done